exercicios/para-casa/sistema_de_alunas_por_floracanabrava.py

```diff
@@ -0,0 +1,157 @@
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 6).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados da aluna: ")
+    nome = input("Nome da aluna: ")
+    sobrenome = input("Agora, informe o sobrenome da aluna: ")
+    while True:
+        turma = input ("Informe qual a turma da aluna: ")
+        if turma == "Turma A" or turma == "Turma B" or turma == "Turma C":
+            break
+        else:
+            print("Turma inválida. Você deve digitar Turma A, Turma B ou Turma C")
+
+    notas = obter_notas()
+    presenca = obter_presenca()
+    participacao = float(input("Participação da aluna: "))
+
+    salvar_dados_alunas(nome, sobrenome, turma, notas, presenca, participacao)
+    
+    return (nome, sobrenome)
+
+def consultar_lista_alunas():
+
+    if not dataset:
+        print("Ainda não há alunas cadastradas")
+        return
+    else:
+        for chave in dataset.keys():
+            print(f"\nAluna: {chave[0]} {chave[1]}")
+
+def consultar_faltas_aluna():
+    nome = input("Por favor, nos informe o nome da aluna: ")
+    sobrenome = input("Agora informe o sobrenome da aluna: ")
+
+    qtd_faltas = dataset[(nome, sobrenome)]["Presença"].count(False)
+    print(f"A quantidade de faltas de {nome} {sobrenome} é {qtd_faltas}.")
+
+    return qtd_faltas
+
+    
+def consultar_notas_aluna():
+    nome = input("Por favor, nos informe o nome da aluna: ")
+    sobrenome = input("Agora informe o sobrenome da aluna: ")
+
+    valores_notas = dataset[(nome, sobrenome)]["Notas"]
+    print(f"As três notas de {nome} {sobrenome} são {valores_notas[0]}, {valores_notas[1]}, {valores_notas[2]}.")
+
+    return valores_notas
+    
+def consultar_status_aprovacao():
+     nome = input("Por favor, nos informe o nome da aluna: ")
+     sobrenome = input("Agora informe o sobrenome da aluna: ")
+
+     notas = dataset[(nome, sobrenome)]["Notas"]
+     media = float(sum(notas)/len(notas))
+     qtd_faltas = dataset[(nome, sobrenome)]["Presença"].count(False)
+     percentual_de_faltas = (qtd_faltas / 5) * 100
```

Comment:
Apesar de saber que no dataset estamos trabalhando com 5 aulas, poderiamos trabalhar com o tamanho da lista, caso haja inclusão de uma aula no nosso dicionário

**Sugestão:**
```suggestion
     percentual_de_faltas = (qtd_faltas / len(dataset[chave]["Presença"])) * 100
```